backend/globaleaks/handlers/submission.py

```diff
@@ -264,11 +268,9 @@ def post(self):
         Response: SubmissionDesc (Token)
         Errors: ContextIdNotFound, InvalidInputFormat, SubmissionFailFields
 
-        This creates an empty submission for the requested context,
-        and returns submissionStatus with empty fields and a Submission Unique String,
-        This is the unique token used during the submission procedure.
-
-        This create a Token, require to complete the submission later.
+        This API create a Token, a temporary memory only object able to keep track of
+        the submission. If the box is under stress, complete the submission will require
+        some effort (hashcash, captcha) kept track in the Token.
```

Comment:
maybe the box should be called always system to use something that may recall to a computer?